src/libs/actions/IOU.js

```diff
@@ -196,14 +196,22 @@ function buildPayPalPaymentUrl(amount, submitterPayPalMeAddress, currency) {
  * @param {String} params.currency
  * @param {String} [params.submitterPhoneNumber] - used for Venmo
  * @param {String} [params.submitterPayPalMeAddress]
+ * @param {String} [params.comment]
  */
 function payIOUReport({
-    chatReportID, reportID, paymentMethodType, amount, currency, submitterPhoneNumber, submitterPayPalMeAddress,
+    chatReportID, reportID, paymentMethodType, amount, currency, submitterPhoneNumber, submitterPayPalMeAddress, comment,
 }) {
     Onyx.merge(ONYXKEYS.IOU, {loading: true, error: false});
+    const newIOUReportDetails = {
+        amount,
+        currency,
+        requestorAccountID: 0, // get the real id
```

Comment:
@nickmurray47 What exactly is the `requestorAccountID` here, please? Should I pass it from the  `IOUModal`?

Reply:
`requestorAccountID` here is the person receiving the payment and it should be passed down from `IOUModal`. I think the report participants would be a good place to get this info.

Reply:
@nickmurray47 I have looked into this but I was unable to find any accountIDs for other than the logged user, thus in this case the `sender`. I think we do not send (or save the accountIDs) for other participants yet.

Reply:
I guess we will need to update some Auth commands too it seems. The `personalDetails` are fetched using [this query](https://github.com/Expensify/Auth/blob/a963f9e27f852913cf8f2f2cf176c8a56b6d6847/auth/lib/Account.cpp#L1712-L1731) in Auth and that return NVP called `personalDetails`. 

In general I am not sure if we return the `accountID` before the report is created (now we need the ID before we create the report). What are your thoughts on this @nickmurray47? I might be missing something

Reply:
Ahhh got it, we don't save the accountIDs but we do save the participant's email, yes?

So I think we need to just pass the recipient email here instead of the accountID and then in the `payIOU` Auth command we will convert the email to the accountID like we do [here](https://github.com/Expensify/Auth/blob/a963f9e27f852913cf8f2f2cf176c8a56b6d6847/auth/command/GetIOUReport.cpp#L18-L21).

How does that sound?

Reply:
That sounds great to me! I am onto it!

Reply:
@nickmurray47 Should we rename it though so it does not say ID but says email same as in case of the `debtorEmail`?

Reply:
Yes! I think that would be appropriate -> `requestorEmail`